modules/display.js

```diff
@@ -0,0 +1,48 @@
+import notDuplicate from './notDuplicate.js';
+import MainBooks from './mainBooks.js';
+
+const container = document.getElementById('container');
+const listOfBooks = new MainBooks();
+
+if (localStorage.book) {
+  const local = JSON.parse(localStorage.book);
+  listOfBooks.array = local;
+}
+
+export default function display() {
```

Comment:
- [ ] Kindly update this function declaration to use arrow function syntax instead of the `function` keyword.